frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -43,24 +44,30 @@ end
 local footerTextGeneratorMap = {
     empty = function() return "" end,
     frontlight = function()
-        if not Device:hasFrontlight() then return "L: NA" end
+        if not Device:hasFrontlight() then return "☼ NA" end
```

Comment:
Huh, it seems odd that this is available as an option at all if `not Device:hasFrontlight()`.

Reply:
Done. Thanks for review :)